instance/tests/integration/integration_instance.py

```diff
@@ -163,6 +163,22 @@ def assert_mongo_db_provisioned(self, instance):
             self.assertTrue(result)
             db.logout()
 
+    @shard(0)
```

Comment:
I'm using `shard 0` here, but if we can add another container, then I can use `shard 4` instead.

Am a little worried about the timing of this one though..  if we end up having to provision 4 appservers total, then it may timeout.